src/index.scss

```diff
@@ -78,15 +71,15 @@ html, body, #root, #app, .full-height{
   text-decoration: none;
 }
 .navigation ul li a{
-  color: var(--dullblue);
+  color: $blue;
 
   border-bottom-style: solid;
-  border-color: var(--dullblueRGBA1);
-  transition: border-color 0.25s;
+  border-color: transparentize($blue, 1);
+  transition: border-color 0.2s;
 }
 .navigation ul li a:hover{
-  border-color: var(--dullblueRGBA2);
-  transition: border-color 0.25s;
+  border-color: transparentize($blue, 0);
```

Comment:
Wow, I didn't know about this function before! That's cool!

Reply:
It's unique to SASS, it's definitely much cleaner than what I was doing